clientapp/src/components/TracerForm.jsx

```diff
@@ -55,7 +56,7 @@ export class TracerForm extends React.Component {
                     <input type="text" value={this.state.city} onChange={this.onCityChange} placeholder="City" className="city" required/>
                     <input type="text" value={this.state.state} onChange={this.onStateChange} placeholder="State" className="state" required/>
                     <input type="text" value={this.state.zip} onChange={this.onZipChange} placeholder="Zip" className="zip" required/>
-                    <Slider startingValue={10} label={"Radius"} unit={"miles"} min={0} max={100} />
+                    <Slider startingValue={10} label={"Radius"} unit={"meters"} min={0} max={100} onChange={this.onRadiusChange} />
```

Comment:
Going up to 10000 makes it hard to user to choose a particular number in between, since we're mapping a large range to a small slider. if we want to give them more flexibility, maybe it should be a regular input box instead of a slider.

Reply:
Should the max={100} be max={100000} , and add some width of the scaler?

Reply:
This is from previous iteration.

Reply:
Ok, it looks good to me.

Reply:
Ok, looks like it's fine, just need reduce master conflict, then this is good.